akka-http-core/src/main/scala/akka/http/javadsl/ConnectionContext.scala

```diff
@@ -41,8 +41,12 @@ object ConnectionContext {
       OptionConverters.toScala(clientAuth),
       OptionConverters.toScala(sslParameters))
 
-  /** Used to serve HTTPS traffic. */
-  // for binary-compatibility, since 2.4.7
+  /**
+   * Used to serve HTTPS traffic.
+   *
+   * @deprecated since 2.4.7
+   */
+  @Deprecated
```

Comment:
Shouldn't we just remove it? I.e. how likely is it that it was more recently chosen by javac's overload resolution?